src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+export const OptionsBitmap = {
+    /** Command execution control - see 1.6.5.8.1. */
+    ExecuteIfOff: BitFlag(0), // TODO default: true.
+
+    /** level effect on color - see 1.6.5.8.2 */
+    CoupleColorTempToLevel: BitFlag(1),
+};
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.1 */
+const  MoveToLevelCommand = TlvObject({
```

Comment:
MoveToLevelCommandRequest ?
This is no the definition of the command but only the request or parameters so the name is not accurate.

Same below